apps/api/src/routes/application-router.ts

```diff
@@ -135,4 +136,68 @@ applicationRouter.get(
 	},
 );
 
+applicationRouter.post('/applications/approve', jsonParser, async (req, res) => {
+  
+	const { applicationId, approverId }: { applicationId?: number; approverId?: number } = req.body;
+
+	if (typeof applicationId !== 'number' || typeof approverId !== 'number') {
+		res.status(400).send({
+		  message: 'Invalid request. Both applicationId and approverId are required and must be numbers.',
+		  errors: 'MissingOrInvalidParameters',
+		});
+		return;
+	  }
+
```

Comment:
Do you have the Prettier extension installed? There are whitespace changes here which don't match what I'd expect to see